it/uk/gov/hmrc/nrs/retrieval/NrsRetrievalIntegrationSpec.scala

```diff
@@ -223,17 +223,17 @@ class NrsRetrievalIntegrationSpec
     val requestWithHeader = wsClientWithXApiKeyHeader(url)
     val retrievalPath = "/retrieval/submission-bundles/vaultId/archiveId"
 
-    def givenSubmissionBundlesReturns(status: Int, withHeaders: Boolean): Unit =
+    def givenSubmissionBundlesReturns(status: Int, withHeaders: Boolean, body: String = "some bytes"): Unit =
       wireMockServer.addStubMapping(
         mappingBuilder(get(urlPathMatching(retrievalPath)), withHeaders)
-          .willReturn(aResponse().withStatus(status)).build())
+          .willReturn(aResponse().withStatus(status).withBody(body)).build())
 
     Seq(
```

Comment:
This spells out the new behaviour. The endpoint will no longer return `OK` when there is an issue upstream.